start/roadmap.md

```diff
@@ -124,4 +124,5 @@ and our facilitation team will be happy to discuss your individual case.
 
 # Getting Help 
 
-For assistance or questions, please email the OSG Facilitation team  at <mailto:support@opensciencegrid.org> or visit the [help desk and community forums](http://support.opensciencegrid.org).
+See our [Get Help page](12000084585) for all the different ways you can reach the OSG Facilitation team. Our goal is 
+to enable you to scale up your computing, so we want to hear from you if you have any questions about using OSG Connect. 
```

Comment:
I think this is fine, but I'd consider replacing OSG Connect because some users of our documentation do not use OSG Connect. Maybe consider replacing with "Open Science Pool resources." Lauren would be better at deciding which term to use. 

As an idea, you could also make the sentence more broad. For example, "Our purpose is to assist you with achieving your computational goals, so please contact us with any questions about using Open Science Pool resources." This is just an idea - I think what you have is perfectly fine.

Reply:
I like this! I'll make the change.